specification/iotcentral/data-plane/Microsoft.IoTCentral/preview/2019-10-28-preview/iotcentral.json

```diff
@@ -1155,9 +1451,6 @@
               "$ref": "#/definitions/RoleCollection"
             }
           }
-        },
-        "x-ms-pageable": {
-          "nextLinkName": null
```

Comment:
Why do you remove this? For what I see "RoleCollection" has nextLink/value as expected for a pageable operation. Seems inconsistent to me.